src/components/DetailsInputs/detailsInputs.util.js

```diff
@@ -79,10 +79,18 @@ export const generateInputsTabContent = (artifacts, showArtifact) => {
         template: (
           <>
             <CopyToClipboard textToCopy={artifact.ui.inputPath} tooltipText="Copy path" />
-            <RoundedIcon tooltipText="Show Details" id="show-details">
-              <Link target="_blank" to={artifact.ui.artifactLink}>
+            <RoundedIcon
+              tooltipText="Show Details"
+              id="show-details"
+              disabled={!artifact.ui?.inputPath.includes('store')}
+            >
```

Comment:
please use const: !artifact.ui?.inputPath.startsWith(MLRUN_STORAGE_INPUT_PATH_SCHEME)

and we need an additional condition to disable preview when request for fetchArtifacts fails (66 line in DetailsInput.js)

Reply:
@Taras-Hlukhovetskyi 
1. const used
2. When a fetch artifacts fails we show "no data ..." so there's no table and preview icon